src/main/java/com/google/api/codegen/transformer/java/JavaSurfaceNamer.java

```diff
@@ -33,6 +33,7 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
+1 - Generally, a lot of apache commons functionality is also available in Google Guava, so it's silly to split our usage across both.